admin_guide/manage_nodes.adoc

```diff
@@ -245,27 +245,45 @@ To list pods that will be migrated without actually performing the evacuation,
 use the `--dry-run` option:
 
 ----
-$ oadm manage-node <node1> <node2> \
-    --evacuate --dry-run [--pod-selector=<pod_selector>]
+$ oadm drain <node1>  --dry-run=true
 ----
 
+If `dry-run` is set to `true`, you only print the object that would be sent,
+without sending it.
+
 To actually evacuate all or selected pods on one or more nodes:
```

Comment:
`To actually evacuate all pods on one or more nodes:` (currently there is no option to select pods)